WSREP_VERSION

```diff
@@ -0,0 +1,2 @@
+WSREP_VERSION_API=25
+WSREP_VERSION_PATCH=28
```

Comment:
In `cmake/wsrep.cmake`, we have set `WSREP_PATCH_VERSION=57`, but it is 28 here. This causes inconsistencies in the versioning. Could you please fix it?